src/configurations/adknowledgeportal/routesConfig.ts

```diff
@@ -239,33 +240,35 @@ const routes: GenericRoute[] = [
     ],
   },
   {
-    to: 'Explore',
-    isNested: true,
+    path: 'Explore',
     routes: [
       {
-        isNested: true,
-        to: 'Programs',
-        synapseConfigArray: [
+        path: 'Programs',
+        routes: [
           {
-            name: 'RouteControlWrapper',
-            isOutsideContainer: true,
-            className: 'ProgramCardList',
-            props: {
-              ...RouteControlWrapperProps,
-              synapseConfig: {
-                name: 'CardContainerLogic',
+            path: '',
+            exact: true,
+            synapseConfigArray: [
+              {
+                name: 'RouteControlWrapper',
+                isOutsideContainer: true,
+                className: 'ProgramCardList',
```

Comment:
The /Explore/Programs page is a good example of the new way to render content on a page that had child routes.

Before, Programs had a `synapseConfigArray`, which rendered the RouteControlWrapper and CardContainerLogic query view. Programs also had `routes` which rendered the details page.

After these changes, we can only have `synapseConfigArray` or `routes`, but not both. So now Programs has `routes`, and the previous synapseConfigArray is included in a route with an empty string path. This new route can be shown/hidden on child pages by specifying `exact`, so this gives us a little more flexibility while having closer behavior to react-router.

Reply:
This explanation was very helpful